recipes/pvxs/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cat <<EOF > configure/RELEASE.local
+EPICS_BASE=${EPICS_BASE}
+EOF
+
+make install
+
+install -d ${PREFIX}/include/pvxs ${PREFIX}/bin/ ${PREFIX}/lib/ ${PREFIX}/epics/modules/pvxs/dbd
+install bin/${EPICS_HOST_ARCH}/* ${PREFIX}/bin/
+install lib/${EPICS_HOST_ARCH}/libpvxs.* ${PREFIX}/lib/
```

Comment:
You probably need to install  libpvxsIoc libary as well to fix the overlinking issue in the build. So,either install the files explicitly or move the wildcard slightly to capture "libpvxs*" instead of "libpvx.*".

I am not sure why we need to run explicit "install" commands on these files. I would expect "make install" to install into $PREFIX. :thinking:

Reply:
Make install would install pvxs, the epics way, inside O.{ARCH} directories. So we need to copy to standard directories if we need them there.